eslint.config.mjs

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+import eslint from "@eslint/js";
+import tseslint from "typescript-eslint";
+import pluginReact from "eslint-plugin-react";
+import eslintPluginPrettierRecommended from "eslint-plugin-prettier/recommended";
```

Comment:
In https://github.com/alveusgg/alveusgg/blob/main/apps/website/eslint.config.js we're using eslint-config-prettier instead of just the plugin -- should we be using that here as well?